packages/shared/src/components/modals/ClickbaitShieldModal.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { type ReactElement } from 'react';
+import { Modal, type ModalProps } from './common/Modal';
+import { Image } from '../image/Image';
+import {
+  Typography,
+  TypographyColor,
+  TypographyType,
+} from '../typography/Typography';
+import { PlusUser } from '../PlusUser';
+import { Button, ButtonVariant } from '../buttons/Button';
+import { webappUrl } from '../../lib/constants';
+import { DevPlusIcon } from '../icons';
+import { usePlusSubscription, useViewSize, ViewSize } from '../../hooks';
+import { LogEvent, TargetId } from '../../lib/log';
+import { Switch } from '../fields/Switch';
+import { useLazyModal } from '../../hooks/useLazyModal';
+
+const ClickbaitShieldModal = (props: ModalProps): ReactElement => {
+  const { logSubscriptionEvent } = usePlusSubscription();
+  const { closeModal } = useLazyModal();
+  const isMobile = useViewSize(ViewSize.MobileL);
+  if (!isMobile) {
+    closeModal();
+    return null;
+  }
+
+  return (
+    <Modal {...props} isDrawerOnMobile>
+      <Image
+        className="mb-5 rounded-16"
+        src="https://daily-now-res.cloudinary.com/image/upload/s--GWqpMG8r--/f_auto/v1732802237/Streak_together_with_a_friend_1_1_pwoill"
```

Comment:
Let's add this to cloudinary list export :)

Reply:
Moved